tests/unit/conf/global.json

```diff
@@ -30,11 +30,21 @@
     "enabled": false,
     "excluded_iocs": {
       "ip": [
-        "127.0.0.0/8",
-        "10.0.0.0/8",
-        "172.16.0.0/12",
-        "192.168.0.0/16",
-        "52.52.52.52/32"
+        {
```

Comment:
even if we need to add keys in the future (one of the considerations i had when expanding this into a hash), we could have the value of the hash be another hash with metadata

Reply:
👍 to this! an added benefit of being a dict is that we can do keyed lookups of the indicator now. previously, this would check for a value in a _list_ which is significantly slower if they list is large